### PR TITLE
Fetching Data in React: add instructions for the browser refresh

### PR DESCRIPTION
**Pull request by ManonLef**
## Because

- since the component is already mounted, and the API data `useEffect()` has an empty dependency array, you'll need to refresh your browser window to see the white screen the lesson mentions. 

## This PR

- add mention of refreshing the browser window. 

## Issue

Closes #XXXXX

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

File: react/the_react_ecosystem/fetching_data_in_react.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ export default Image;
 
 Working over the network is inherently unreliable. The API you're making a request to might be down, there could be network connectivity issues, or the response you receive could contain errors. A multitude of things can go wrong, and if you don't preemptively plan for errors, your website can break or appear unresponsive to users.
 
-To simulate a network error, scroll up to the previous code snippet and change the `fetch` URL to something random. The page will remain a blank white screen, without giving the user any indication that the page has finished loading or that there was an error.
+To simulate a network error, scroll up to the previous code snippet and change the `fetch` URL to something random. After a refresh of your browser window, the page will remain a blank white screen, without giving the user any indication that the page has finished loading or that there was an error.
```
Comment (ManonLef) [inline react/the_react_ecosystem/fetching_data_in_react.md:72]:
```suggestion
To simulate a network error, scroll up to the previous code snippet and change the `fetch` URL to something random. After a refresh of your browser window, the page will remain a blank white screen, without giving the user any indication that the page has finished loading or that there was an error.
```

**Review by wise-king-sullyman — APPROVED** [review]
One thought, rather than adding this note we could make the url a variable and add it to the dependency array to automate the process.

Not opposed to just adding the note either though, just wanted to voice that alternative, feel free to merge if there's a reason to not do the above that I'm just not thinking of.

**Comment by ManonLef** [comment]
You're probably right @wise-king-sullyman ! I am still finding my way through React. I'll make sure to rewrite and test this probably somewhere next week. I think I approached it with url stored in a variable in my own project too actually way after opening this :D. Thanks!

**Review by kamranzafar4343 — COMMENTED** [review]
just reviewing the code

**Comment by thatblindgeye** [comment]
@wise-king-sullyman @ManonLef would we want to merge this in with the note for now?

**Comment by ManonLef** [comment]
I forgot about this, thanks for reviving it @thatblindgeye ! I will merge it

**Pull request merged**